ui/src/components/useMessageDialog.tsx

```diff
@@ -117,12 +118,12 @@ const MessageDialogContainer = () => {
     const subject = `${messageDialogParameters.current_user.first_name} ${messageDialogParameters.current_user.last_name} connected with you`;
     const body = `Message from ${messageDialogParameters.current_user.first_name} ${messageDialogParameters.current_user.last_name}:\n${message}`;
     JobsGoWhereApiClient.post(`${process.env.REACT_APP_API}/sendmessage`, {
-      to: receiverId, 
-      subject, 
-      body}, {
-        headers : {'Content-Type' : 'application/json'}
-      }).then(res => {
-      console.log(res);
+      to: receiverId,
+      subject,
+      body,
+    }).then((res) => {
+      toast("👍 Good Job! Message Sent! Check your email for replies.");
+      showMessageDialog(false);
```

Comment:
Can you add a toast on error too? The flow breaks when the API returns error.

Reply:
Good idea. Added toast message in request catch block

![image](https://user-images.githubusercontent.com/3888250/93786626-1f479180-fc62-11ea-8475-6e9d0a83abfd.png)